File: Data Structures & Algorithms/height-checker/submission-0.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        std::vector<int> sorted = heights;
        std::sort(sorted.begin(), sorted.end());
        int count = 0;
        for (int i = 0; i < heights.size(); i++) {
            if (sorted[i] != heights[i]) {
                count++;
            }
        }
        return count;
    }
};

File: Data Structures & Algorithms/sort-the-people/submission-0.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        tab = {}
        for i in range(len(names)):
            tab[heights[i]] = names[i]
        heights.sort(reverse = True)
        ans = []
        for h in heights:
            ans.append(tab[h])
        return ans